packages/components/src/components/popover/popover-wrapper.jsx

```diff
@@ -0,0 +1,125 @@
+                        arrowColor={has_error ? 'var(--status-danger)' : 'var(--general-active)'}
+                        arrowSize={5}
+                    >
+                        <div
+                            id={id}
+                            className={classNames(
+                                classNameBubble,
+                                'dc-popover__bubble',
+                                { 'dc-popover__bubble--error': has_error },
+                            )}
+                        >
+                            { !disable_message_icon && icon === 'info' &&
+                                <i className='dc-popover__bubble__icon'>
+                                    <IconInfoBlue />
+                                </i>
+                            }
+                            <span className='dc-popover__bubble__text'>
+                                { message }
+                            </span>
+                        </div>
+                    </ArrowContainer>
+                )}
+            >
+                <div
+                    className={classNames('dc-popover', className)}
+                    id={id}
+                    onMouseEnter={this.toggleOpen}
+                    onMouseLeave={this.toggleClose}
+                >
+                    <div className={classNames(classNameTarget, 'dc-popover__target')}>
+                        {!disable_target_icon &&
+                        <i className={message ? 'dc-popover__target__icon' : 'dc-popover__target__icon--disabled'}>
+                            {(icon === 'info')     && <IconInfoOutline className={icon_class_name} />}
+                            {(icon === 'question') && <IconQuestion className={icon_class_name} />}
+                            {(icon === 'dot')      && <IconRedDot className={icon_class_name} />}
+                            {(icon === 'counter')  && <span className={icon_class_name}>{ counter }</span>}
+                        </i>
+                        }
+
+                        {children}
+                    </div>
+                </div>
+            </Popover>
+        );
+    }
+}
+
+PopoverWrapper.defaultProps = {
+    portal_container: 'deriv_app',
+    margin          : 0,
+};
+
+PopoverWrapper.propTypes = {
+    alignment           : PropTypes.string,
+    children            : PropTypes.node,
+    className           : PropTypes.string,
+    classNameBubble     : PropTypes.string,
+    classNameTarget     : PropTypes.string,
+    classNameTargetIcon : PropTypes.string,
+    counter             : PropTypes.number,
+    disable_message_icon: PropTypes.bool,
+    disable_target_icon : PropTypes.bool,
+    has_error           : PropTypes.bool,
+    icon                : PropTypes.string,
+    id                  : PropTypes.string,
+    margin              : PropTypes.number,
+    message             : PropTypes.string,
+    portal_container    : PropTypes.string,
+};
+
+export default PopoverWrapper;
```

Comment:
can we keep this component name as `Popover` ? 😄

Reply:
Had to change the name to that because the plugin we are using is also called a `Popover` and it would try to use itself and it just breaks.

Reply:
Agreed with @kelly-binary , we should keep the name the same as it's a component that will be used in the end. Maybe you can try to use `import` as / aliasing?

Since `Popover` is already the default export from `react-tiny-popover` you have two options: -
1) `import TinyPopover, { ArrowContainer } from 'react-tiny-popover'; and use `TinyPopover` instead.
2) `import { default as TinyPopover, ArrowContainer } from 'react-tiny-popover;` and use `TinyPopover` instead.

Reply:
I made changes and used your second suggestions @zaki-binary